_gsocproposals/2020/proposal_MCnetSherpa.md

```diff
@@ -0,0 +1,50 @@
+---
+title: MCnet/SHERPA - Accelerating Monte Carlo simulations for the LHC
+layout: gsoc_proposal
+project: MCnet
+year: 2020
+organization:
+  - UDUR
+---
+
+# Description
+
+SHERPA is a Monte Carlo event generator for collider-based particle physics experiments such as the ones perfomed at CERN's Large Hadron Collider (LHC).  It simulates full events, starting with the incident protons and finishing with the intricate patterns of the produced high-energetic particles that will eventually interact with the detector, thereby producing the indispensible link between theory and data, and allowing the interpretation of the latter in the light of the former.  To describe such events a variety of techniques and models must be invoked, many of which are based on the perturbative expansion of Quantum Field Theory.  This is true in particular for the starting point of the simulations where the so-called hard process defines the overall structure of the events.  To describe this central bit of the simulation, cross sections must be evaluated as integrals of the corresponding transition amplitudes over the multi-dimensional phase space of the outgoing particles.  Due to the high dimensionality of the phase space, this integration is achieved through Monte Carlo methods.
+
+In the ongoing quest for increased precision of our understanding and modelling of the physics involved in the proton collisions, ever higher orders of the perturbation expansion must be evaluated.  This results in ever more complicated expressions for the transition amplitudes and an ever increasing number of final state particles and therefore the dimensionality of the phase space integration.  It is therefore not surprising that this first step of event simulation is the most intensive source of CPU consumption and becoming the bottleneck for the full exploitation of future LHC results.

```

Comment:
"first step of event simulation" => "first step of the event simulation" or "first step of event simulations"